Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -241,7 +242,24 @@ bool ES::LaunchTitle(u64 title_id, bool skip_reload)
   // (supposedly when trying to re-open those files).
   DiscIO::NANDContentManager::Access().ClearCache();
 
-  if (IsTitleType(title_id, IOS::ES::TitleType::System) && title_id != TITLEID_SYSMENU)
+  u32 device_id;
+  constexpr u32 DEFAULT_DEVICE_ID = 0x0403ac68;
```

Comment:
We do, in ec_wii, but in the implementation file. I thought about exposing it in a header, but I'm not sure about doing so as it's a detail that shouldn't be public and the ID isn't going to change. What do you think?